infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -165,4 +191,23 @@ private KubernetesRoleBinding createExecRoleBinding() {
                 .build())
         .build();
   }
+
+  private KubernetesRoleBinding createCustomRoleBinding(String clusterRoleName) {
```

Comment:
Should not `cluster` be used here instead of `custom`?

Reply:
I decided to use `createCustomRoleBinding` since we're creating a role binding for the cluster role here. 

Note that role bindings can bind both Roles and ClusterRoles